src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -482,12 +496,11 @@ export default class WaterfallPlot<D extends IBaseWaterfallPlotData> extends Rea
                             domainPadding={PLOT_DATA_PADDING_PIXELS}
                             singleQuadrantDomainPadding={false}
                         >
-                            {this.title}
                             {this.legend}
                             {this.props.horizontal && <VictoryAxis
                                 domain={this.plotDomainX}
                                 orientation="top"
-                                offsetY={80}
+                                offsetY={30}
```

Comment:
I know that it was hard coded before, but better extract this as a constant at the top of the screen under a clear name.